helix-view/src/document.rs

```diff
@@ -121,6 +121,9 @@ pub struct Document {
 
     diagnostics: Vec<Diagnostic>,
     language_server: Option<Arc<helix_lsp::Client>>,
+
+    // when document was used for most-recent-used buffer picker
+    pub used_at: std::time::SystemTime,
```

Comment:
SystemTime is slower and not monotonic. https://doc.rust-lang.org/std/time/struct.Instant.html

Reply:
done. `Instant` used instead of `SystemTime`